_posts/2017-09-08-web.md

```diff
@@ -12,9 +12,84 @@ hierarchy:
 ---
 
 
-## Web apps
+## An Introduction 
 
-TODO
+Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to provide a successful career in web development.
+
+
+#### How does the Internet work? 
+
+The Internet has become the backbone of our modern-day world, yet few individuals take the time to understand what it is, and how it works. Since it has become such a large part of our lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
+
+This section will explain the underlying infrastructure and technologies that make the Internet work; it won't provide a very detailed explanation, and resources will be provided at the end for those who wish to learn more. Any comments, questions, suggestions, etc are welcome and may be directed to the author(s) at [team@hackmit.org](mailto:team@hackmit.org).
+
+Let's take a dive! Let's do a deep dive of exactly what happens, from when you type `https://google.com` in your browser to the moment it shows up on your screen.  
+
+##### Internet Addressing
+
+When the Internet was first founded, it nothing more than a network of computers for academic purposes; there was no way to go to `google.com`, because there was no concept of a named website, or a *domain*. Computers were connected to each other solely on the basis of an address of the form *xxx.xxx.xxx.xxx*, where *x* is a number from *0 - 255*.
+
+> More recently, these addresses can also take a form of *xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx*, where *x* is a [hexadecimal number](https://en.wikipedia.org/wiki/Hexadecimal) between 0 and 15! This hexadecimal notation, commonly referred to as [IPv6](https://en.wikipedia.org/wiki/IPv6), was developed in 2012 after we began running out of simpler IP addresses! 
+
+This address is known as an IP address, and this is what our computer systems still use. When you go to a *domain*, it looks up the underlying IP address, using the *Domain Name System* (DNS), and directs you to there. For example, Google's IP address is `216.58.219.46`. It's important to note that IP addresses are unique; no two servers can share the same IP address! Can you see why this may be a problem?
```

Comment:
Hmm I see a different IP address. Maybe because Google has many IP addresses that change based on location? Might be nice to include a parenthetical, footnote, or something saying this or whatever might be the explanation.

Reply:
change to "Can you see how this _could be_ a problem?". "may be" implies that unique IPs are problematic, but what you mean is that non-unique IPs are problematic